.pre-commit-config.yaml

```diff
@@ -69,3 +69,4 @@ repos:
       - id: flake8
         additional_dependencies:
           - flake8-bugbear == 20.11.1
+          - flake8-init-return == 1.0.0
```

Comment:
is it possible to also use flake8 to add return type hint for `__str__`, `__repr__` and other dunder method in python ?

Reply:
If we write more plugins, sure!